openmls/src/extensions/external_pub_extension.rs

```diff
@@ -0,0 +1,29 @@
+use tls_codec::{TlsDeserialize, TlsSerialize, TlsSize};
+
+use super::{Deserialize, Serialize};
+use crate::prelude::HpkePublicKey;
+
+/// ```c
+/// // draft-ietf-mls-protocol-16
+/// struct {
+///     HPKEPublicKey external_pub;
+/// } ExternalPub;
+/// ```
+#[derive(
+    PartialEq, Eq, Clone, Debug, Serialize, Deserialize, TlsSerialize, TlsDeserialize, TlsSize,
+)]
+pub struct ExternalPubExtension {
+    external_pub: HpkePublicKey,
+}
+
+impl ExternalPubExtension {
+    /// Create a new `external_pub` extension.
+    pub fn new(external_pub: HpkePublicKey) -> Self {
+        Self { external_pub }
+    }
+
+    /// Get the HPKE public key.
```

Comment:
"a reference to"
It's not returning the key itself.

Reply:
I took care of this.